libraries/Microsoft.Bot.Builder.AI.QnA/Utils/GenerateAnswerUtils.cs

```diff
@@ -143,6 +143,11 @@ private static void ValidateOptions(QnAMakerOptions options)
             {
                 options.MetadataBoost = new Metadata[] { };
             }
+
```

Comment:
Where did this come from?

Reply:
no idea, it's in the 4.7 branch...

---
In reply to: [375486469](https://github.com/microsoft/botbuilder-dotnet/pull/3320#discussion_r375486469) [](ancestors = 375486469)